src/containers/DidView/DidView.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react'
+import { Contact } from '../../types/Contact'
+import './DidView.scss'
+import Code from '../../components/Code/Code'
+
+type Props = {
+  did: Contact['did']
+  children: any
+}
+
+const DidView = (did: Props) => {
```

Comment:
Change request:
Please rename this to `DidDocumentView`; what you're displaying really is the DID Doc.

Reply:
I have updated everything to match.